extensions/default/src/Components/CloudServerConfigurationComponent.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { ReactElement, useCallback, useEffect, useState } from 'react';
+import { useModal } from '@ohif/ui';
+
+function CloudServerConfigurationModalComponent({
```

Comment:
Generic component for configuring a cloud server API data source. It uses the generic/base interfaces `BaseCloudServerAPI` and `BaseCloudServerAPIItem`